### DISCUSSION TITLE
Zlibary下载

### DISCUSSION THREAD

**Discussion by ascendho** [tw93/Pake#262] [Q&A]
第一个问题，今天试了下下载电子书，发现很多下载链接打不开或者很慢，请问需要木弟子吗？
另外一个问题，这个zlibray电子书库存是实时更新的吗还是静态拷贝的呢，谢谢。

**Comment by tw93** [comment]
有4个可选的地址，可以都试试

**Reply by qwt234** [reply]
非常感谢您！！zlibrary的Mac端在我这里打开是白板一块，什么都没有。是不是我系统版本低的原因？我的是10.14.2。小白一个。
<img width="1200" alt="image" src="https://user-images.githubusercontent.com/123139168/213613170-b5c8f515-8406-4f50-b457-2b7849876ac7.png">

**Reply by tw93** [reply]
看看是不是地址原因 你再关掉重开试试 或者先试一下科学上网 按理不需要科学

**Reply by ManonBezos** [reply]
我也是下载了Mac版，打开一片空白。然后又下了windows也不能用。爬梯子和不爬梯子都是一片空白
<img width="1186" alt="image" src="https://user-images.githubusercontent.com/123224608/213843411-70d4e0f9-825a-4134-a6de-42b9b6718ded.png">

**Reply by qwt234** [reply]
> 看看是不是地址原因 你再关掉重开试试 或者先试一下科学上网 按理不需要科学

都试过了，也是白板。我问过其他人，版本高些的系统打开就有。我觉得可能是系统版本的缘故。不知道您那边能否测试一下究竟是什么原因。

**Comment by mylespjf** [comment]
在访达里右键，在弹出的菜单栏里点打开

2023年1月21日 12:20，ManonBezos ***@***.***> 写道：



我今天下载了mac版的zlibrary, 但是无法安装，显示无法验证开发者，请问应该如何解决这个问题？
[image]<https://nam12.safelinks.protection.outlook.com/?url=https%3A%2F%2Fuser-images.githubusercontent.com%2F123224608%2F213843086-b43560b0-fc17-490d-891b-f24903f2dec2.png&data=05%7C01%7C%7C043a0dd345e442e2814008dafb66dbc4%7C84df9e7fe9f640afb435aaaaaaaaaaaa%7C1%7C0%7C638098716453227554%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=%2FzDwjYTqxmKXyMdry4H8Z6BrJMAnCgCLDtZAzHT5hE0%3D&reserved=0>

**Reply by ManonBezos** [reply]
开发者验证搞定了，就是zlibrary 安装好，即便从访达右键打开一就是一片空白。
<img width="1232" alt="image" src="https://user-images.githubusercontent.com/123224608/213846982-77c71946-69a8-4eab-b36a-f4b7a696b04f.png">

**Comment by mylespjf** [comment]
是不是还得把科学上网打开啊😄

2023年1月21日 14:31，ManonBezos ***@***.***> 写道：



开发者验证搞定了，就是zlibrary 安装好，即便从访达右键打开一就是一片空白。
[image]<https://nam12.safelinks.protection.outlook.com/?url=https%3A%2F%2Fuser-images.githubusercontent.com%2F123224608%2F213846982-77c71946-69a8-4eab-b36a-f4b7a696b04f.png&data=05%7C01%7C%7C7fa333ca0db644db82e208dafb79150e%7C84df9e7fe9f640afb435aaaaaaaaaaaa%7C1%7C0%7C638098794711631337%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=GGASGkpMr7drlqSrypnm45redZ6YCCBZqdWWBitYV90%3D&reserved=0>

**Comment by ManonBezos** [comment]
科学上网一直开着呀，不然怎么用推特嘛。新年快乐哦！

发自我的 iPhone

在 2023年1月21日，15:26，qwt234 ***@***.***> 写道：

﻿

看看是不是地址原因 你再关掉重开试试 或者先试一下科学上网 按理不需要科学

都试过了，也是白板。我问过其他人，版本高些的系统打开就有。我觉得可能是系统版本的缘故。不知道您那边能否测试一下究竟是什么原因。

**Comment by Megadotzip** [comment]
大年初一，早上Zlib不翻墙无法正常打开（Win11）

**Reply by tw93** [reply]
看起来是一个问题，计划部署一个国内的版本

**Comment by fayfeifay** [comment]
我也遇到一块白板的问题了,一直有挂梯子,前一段时间还可以用.不知道是不是因为我电脑版本太低了? 15年的mac air
不过同一台电脑前几天还可以.

**Reply by tw93** [reply]
哦哦 这个最近应该是域名被污染了，重新进去 挂梯子应该可以，最近正在更换

**Reply by tw93** [reply]
已经修复了，重新下载最新的即可

**Comment by qwt234** [comment]
下载了作者最新的zlib包，Mac版。打开依然是白板
<img width="499" alt="image" src="https://user-images.githubusercontent.com/123139168/215237906-d89e5379-cc97-4081-b24f-7c7d584c329d.png">

**Reply by qwt234** [reply]
<img width="244" alt="image" src="https://user-images.githubusercontent.com/123139168/215237971-fccd1418-3c6c-49ec-8a3b-33633af0628a.png">

**Reply by qwt234** [reply]
一直眼馋，用不上啊！！

**Reply by tw93** [reply]
有些奇怪 我这边看着是OK的了 重新退出试试看呢

**Reply by tw93** [reply]
此外可以试试 微信读书是否可以打包，我排查下是页面问题还是框架问题

**Reply by qwt234** [reply]
> 此外可以试试 微信读书是否可以打包，我排查下是页面问题还是框架问题
微信读书打开是正常的。zlib老样子，依然白版。试了好多次了。
<img width="854" alt="image" src="https://user-images.githubusercontent.com/123139168/215299304-d3c935a3-3b38-471c-b0d8-3a9de95f9a83.png">

**Comment by ManonBezos** [comment]
大神们，这个问题cydiar给我解决了，其实是梯子的问题，有些梯子禁了这个弹跳。
梯子换全局模式就能看桌面的zlibrary了。

2023年1月28日 11:56，Tw93 ***@***.******@***.***>> 写道：



此外可以试试 微信读书是否可以打包，我排查下是页面问题还是框架问题

**Comment by Mr-Sherlock** [comment]
下载了 windows版 显示提示安装需要地址在硬件上 但是我的地址在硬件上啊
